dev_docs/07_visual_testing_guide.md

```diff
@@ -0,0 +1,93 @@
+# Visual Testing
```

Comment:
This file should be referenced in the getting started file that is the entry point for this part of the documentation. https://github.com/KshitijThareja/kolibri-design-system/blob/6dc4cab441cac3a76fc438a217e03199775fbf74/dev_docs/01_getting_started.md#L58